File: bridge/java/TextMessageSender.java
package Bridge;

// UML diyagramındaki ConcreteImplementor (TextMessageSender) kısmına denk gelmektedir.
// Implementor arayüzünü (IMessageSender) uygular.
public class TextMessageSender implements IMessageSender {
    @Override
    public void sendMessage(Message message) {
        System.out.println("TextMessageSender: Sending text message...");
    }
}

File: bridge/kotlin/TextMessageSender.kt
package Bridge

// UML diyagramındaki ConcreteImplementor (TextMessageSender) kısmına denk gelmektedir.
// Implementor arayüzünü (IMessageSender) uygular.
class TextMessageSender : IMessageSender {
    override fun sendMessage(message: Message) {
        println("TextMessageSender: Sending text message...")
    }
}
